docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -269,7 +269,7 @@ In YAML:
           ansible_host: 192.0.2.50
 
 In the above example, running Ansible against the host alias "jumper" will connect to 192.0.2.50 on port 5555.
-This only works for hosts with static IPs, or when you are connecting through tunnels.
+This can be useful when you are connecting through tunnels.
```

Comment:
This should probably be further clarified that `ansible_host` can be anything resolvable by a connection plugin. This could be an IP address, an FQDN, or a path in the case of something like the `chroot` connection plugin.

Reply:
Good point ! I simply pointed to the relevant section (of the same page!) to avoid duplication in e9c8cf2.